packages/contracts-bedrock/src/L1/SystemConfig.sol

```diff
@@ -484,6 +492,20 @@ contract SystemConfig is ProxyAdminOwnedBase, OwnableUpgradeable, Reinitializabl
         _resourceConfig = _config;
     }
 
+    /// @notice Toggles a feature flag. Can only be called by the ProxyAdmin or its owner.
+    /// @param _feature Feature to toggle.
+    /// @param _enabled Whether the feature should be enabled or disabled.
+    function toggleFeature(bytes32 _feature, bool _enabled) external {
+        // Features can only be toggled by the ProxyAdmin or its owner.
+        _assertOnlyProxyAdminOrProxyAdminOwner();
+
+        // Toggle the feature.
+        isFeatureEnabled[_feature] = _enabled;
+
+        // Emit an event.
+        emit FeatureToggled(_feature, _enabled);
+    }
```

Comment:
Nit, would either do `setFeature(feature, bool)` or `toggleFeature(feature)` or `enableFeature()/disableFeature()`, since this method currently doesn't force a toggle. Maybe worth preventing no-op state changes to minimize the chance of mistakes?